products/sle/main.pm

```diff
@@ -886,10 +886,11 @@ elsif (get_var('LIBSOLV_INSTALLCHECK')) {
 elsif (get_var("EXTRATEST")) {
     boot_hdd_image;
     load_extra_tests();
-    loadtest "console/coredump_collect";
-    loadtest "shutdown/cleanup_before_shutdown";
-    loadtest "shutdown/shutdown";
-
+    unless (check_var('EXTRATEST', 'wicked')) {
+        loadtest "console/coredump_collect";
```

Comment:
Why not just `loadtest "console/coredump_collect" if get_var('COLLECT_COREDUMPS');` ?
The variable is already present in the coredump code, if the coredump is not uploaded then it does not make much sense to run the test.

Reply:
what about shutdown tests ? why I need them in wicked tests ?

Reply:
and obviously bind execution of shutdown tests with coredump variable does not make sense

Reply:
If the test suite is not publishing image at all then just drop it

Reply:
> If the test suite is not publishing image then just drop it

 @mimi1vx  - is it ok for you ?

Reply:
> > If the test suite is not publishing image then just drop it
> 
> @mimi1vx - is it ok for you ?

I think the correct shutdown of SUT is a valid part of testsuite. And can catch problems.

> Why not just `loadtest "console/coredump_collect" if get_var('COLLECT_COREDUMPS');` ?
hmm.  maybe. but still, this is nonintrusive and  hasn't  any impact on other tests
> The variable is already present in the coredump code, if the coredump is not uploaded then it does not make much sense to run the test.

NO is part of `post_fail_hook`, not of `collect_coredump` - 

> collect_coredump